modules/metering-install-configure-storage.adoc

```diff
@@ -0,0 +1,215 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-installing-metering.adoc
+
+[id="metering-install-configure-storage_{context}"] 
+= Configuring Metering storage
+
+Configure the metering operator by editing the default metering configuration file created when you install the metering stack. Some configuration options, such as storage, must be configured before installing the metering stack. Use the documentation below and make changes to the `spec` section of the default configuration file.
+
+[NOTE]
+====
+The Configuring Metering documentation covers all available
+configuration options for the Metering Operator.
+====
+
+Metering requires persistent storage in two main ways. First, the data collected
+for reporting and the results of reports is usually kept in some form of object
+storage or distributed file system. You can configure metering to store this
+data locally or in a cloud provider of your choice.
+
+Second, a Hive metastore manages data to improve the efficency and speed of
+metering reports. By default, this information is stored in an embedded
+database called Derby, which keeps it's data on disk in a PersistentVolume. For
+larger deployments, this metadata can also be configured to use an existing MySQL or PostgreSQL database,
+instead of Derby. Read the documentation on configuring the Hive metastore for
+more details.
+
+[id="metering-storing-data-in-amazon-s3_{context}"]
```

Comment:
Recommend to move this (and the other subsections of "Storing data in Microsoft Azure " and "Storing data in Google Cloud Storage") each into their own module file. Then you can include them in at the assembly and increment the leveloffset field to bump the heading to a level 2.